### PR TITLE
Fix home page active and past contests links to corresponding contests

### PR DESCRIPTION
**Pull request by msmner**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/423

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/ContestsList.tsx [hunk 1]
```diff
@@ -14,13 +13,13 @@ import styles from './ContestsList.module.scss';
 interface IContestsListProps {
     title: string;
     contests: IIndexContestsType[];
-    contestState?: ContestStatus;
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/ContestsList.tsx:17]: Rename to contestStatus

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx [hunk 2]
```diff
@@ -21,17 +21,22 @@ const HomeContests = () => {
         })();
     }, [ getForHome ]);
 
+    const contestStatusArr = useMemo(
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx:24]: why is this needed?

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/ContestsList.tsx (hunk 1 above)
Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/ContestsList.tsx:17]: Done.

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx (hunk 2 above)
Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx:24]: I need to pass the contest status enumeration value as integer to the contests list component. Since ContestStatus is a string enumeration, I could not find any other way to convert its string values to integer :/

**Review by mishogv — APPROVED** [review]

**Review by PetarNeshkov — APPROVED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx (hunk 2 above)
Comment (shsimeonova) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx]: Why not just change the type contestStatus?: ContestStatus ? This way the class that is generated in seeAllButtonClassName    will contain a number

Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx]: When I change it to ContestStatus, it just takes the string value.

**Review by shsimeonova — APPROVED** [review]
